### PR TITLE
Fix comment

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/coreclr/gc/gcpriv.h [hunk 1]
```diff
@@ -140,7 +140,7 @@ inline void FATAL_GC_ERROR()
 //
 // This means any empty regions can be freely used for any generation. For
 // Server GC we will balance regions between heaps.
-// For now disable regions for StandAlone GC, NativeAOT and MacOS builds
+// For now disable regions for standalone GC and macOS builds
 #if defined (HOST_64BIT) && !defined (BUILD_AS_STANDALONE) && !defined(__APPLE__)
 #define USE_REGIONS
 #endif //HOST_64BIT && BUILD_AS_STANDALONE
```
Comment (cshung) [inline src/coreclr/gc/gcpriv.h:146]: NIT: Change this line as well?

**Review by mangod9 — APPROVED** [review]

File: src/coreclr/gc/gcpriv.h (hunk 1 above)
Comment (jkotas) [inline src/coreclr/gc/gcpriv.h:146]:
```suggestion
#endif //HOST_64BIT && BUILD_AS_STANDALONE && !__APPLE__
```

**Review by cshung — APPROVED** [review]
Thanks!

**Comment by jkotas** [comment]
/ba-g Timeouts in legs that are not critical to validate this change

**Pull request merged**